BoxContentSDK/BoxContentSDK/Models/BOXUser.h

```diff
@@ -7,12 +7,43 @@
 #import "BOXModel.h"
 #import "BOXEnterprise.h"
 
+@protocol UniqueSDKUser
+
+@property (nonnull,  nonatomic, readonly, strong) NSString *uniqueId;
+@property (nullable, nonatomic, readonly, strong) NSString *name;
+@property (nullable, nonatomic, readonly, strong) NSString *login;
```

Comment:
Maybe add doc comments to the protocol, as well as, or rather than, the ServerAuthUser.

Reply:
These properties (and in ServerAuthUser) should be `copy` rather than `strong`.